src/main/java/github/weichware10/toolbox/AppController.java

```diff
@@ -0,0 +1,89 @@
+package github.weichware10.toolbox;
+
+import java.io.IOException;
+
+import github.weichware10.util.Logger;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ButtonBar.ButtonData;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.scene.control.ButtonBar.ButtonData' import. Should be before 'javafx.scene.control.TextField'.